compiler/rustc_codegen_gcc/build_system/src/test.rs

```diff
@@ -677,7 +677,7 @@ fn test_projects(env: &Env, args: &TestArg) -> Result<(), String> {
 fn test_libcore(env: &Env, args: &TestArg) -> Result<(), String> {
     // FIXME: create a function "display_if_not_quiet" or something along the line.
     println!("[TEST] libcore");
-    let path = get_sysroot_dir().join("sysroot_src/library/core/tests");
+    let path = get_sysroot_dir().join("sysroot_src/library/coretests/tests");
```

Comment:
```suggestion
    let path = get_sysroot_dir().join("sysroot_src/library/coretests");
```
and remove `compiler/rustc_codegen_gcc/patches/0022-core-Disable-not-compiling-tests.patch`.